strapi/strapi
# [DEITS] Fix export for nested components & dynamic zones

### What does it do? Why is it needed?

Fix the entities export for nested components & dynamic zone (in components).
Basically, if a component contains either a component or a dynamic zone attribute, it's currently not populated (stopped at the first level). This PR aims to add nested capabilities to the populate strategy used.

### How to test it?

- In the monorepository getstarted example application, create a restaurant entity with a dz values that contains components & nested components
- Try exporting the content of the app using yarn strapi export --encrypt=false
- In the exported entities, your restaurant should have exported every component level of your dynamic zone

**Waiting for https://github.com/strapi/strapi/pull/14879 to be merged before being able to test this fix**

### Related issue(s)/PR(s)

https://github.com/strapi/strapi/pull/14879
